django/http/request.py

```diff
@@ -230,6 +288,17 @@ def body(self):
         if not hasattr(self, '_body'):
             if self._read_started:
                 raise RawPostDataException("You cannot access body after reading from request's data stream")
+
```

Comment:
I gave up. :) As it is, naked read()s are not protected against slurping a large amount of data into memory, but that seems ok to everyone else.